Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.16 as builder
```

Comment:
use 1.17 as it's available?

Reply:
> use 1.17 as it's available?

This is the default version kubebuilder creates, I'll update the go version here and upgrade the packages in a follow-up pr.

Reply:
changed